src/components/show-products/index.js

```diff
@@ -43,27 +67,46 @@ export const ShowProducts = () => {
 
   return (
     <ContentContainer>
+      <form className="filter-form">
+        <label htmlFor="filter">
+          Filter items
+          <input
+            className="input-item"
+            type="search"
+            id="filter"
+            name="filter"
+            aria-label="Search through list content"
+            placeholder="Start typing here..."
```

Comment:
Note that this reads this way in VoiceOver (Mac OS's Screen reader):

<img width="1038" alt="Screen Shot 2021-11-05 at 11 00 36 AM" src="https://user-images.githubusercontent.com/843075/140532256-033d028d-4d8c-4935-af09-93b9c7fa5861.png">

One minor improvement could be to add an end mark at the end of the `aria-label` value, so this could read better in the SR:

`Search through list content. Start typing here...` 
```suggestion
            aria-label="Search through list content."
            placeholder="Start typing here..."
```